hummingbot/templates/conf_pure_market_making_strategy_TEMPLATE.yml

```diff
@@ -79,8 +79,11 @@ hanging_orders_cancel_pct: null
 # Whether to enable order optimization mode (true/false).
 order_optimization_enabled: null
 
-# The depth in base asset amount to be used for finding top bid and ask (for order optimization mode).
-order_optimization_depth: null
+# The depth in base asset amount to be used for finding top ask (for order optimization mode).
+ask_order_optimization_depth: null
```

Comment:
When adding new parameters to config map, you'll need to increase the template version number, otherwise the bot will not apply new parameters to the old config file.

Reply:
Done!